packages/web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/web-components",
-  "version": "4.55.0",
+  "version": "5.0.0",
```

Comment:
When adding components, this can just be `minor` (removing/breaking changes are major).

https://github.com/department-of-veterans-affairs/component-library?tab=readme-ov-file#minor

![Screenshot 2024-01-18 at 10 16 34 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/7e31cf88-f020-45db-8cd0-798ec48c55fb)